pkg/funky/router.go

```diff
@@ -0,0 +1,199 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+package funky
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+
+	"golang.org/x/sync/semaphore"
+)
+
+// FirstPort the starting port number for servers created by a Router
+var FirstPort uint16 = 9000
```

Comment:
NIT: could this be a constant instead of var?

Reply:
sure